### ISSUE TITLE
[RFE] Introduce a flag to diss M$

### ISSUE THREAD

**Issue by ahmubashshir** [MrGlockenspiel/activate-linux#137]
It'd be awesome to have a flag to show
```
No need to activate Linux

We're not as annoying as Microsoft
```
<sub>I am proposing `-M` flag for this</sub>

**Comment by jeverling** [comment]
I am proposing `-$` flag for this.

**Comment by Ruby-Dragon** [comment]
seems interesting, may look into it

**Comment by Ruby-Dragon** [comment]
should be solved by PR #139
